### ISSUE TITLE
fix bug where Firefox crashes when selection box is dragged too quickly

### ISSUE THREAD

**Issue by dave-doty** [UC-Davis-molecular-computing/scadnano#566]
I noticed this bug while working in this branch: 

https://github.com/UC-Davis-molecular-computing/scadnano/tree/show-hand-grabbing-cursor-when-pointer-is-able-to-grab-for-click-and-drag-panning

which is for implementing issue #55. However, the bug also appears on the current dev branch when running locally. I *cannot* reproduce it on the deployed dev version https://scadnano.org/dev/, only locally, but I'm worried that it might still be a problem in the deployed production version.

Press Shift in select mode, click the left mouse button, and drag the cursor around rapidly to alter the size of the selection box. It causes an error like this in Firefox:

![image](https://user-images.githubusercontent.com/19274365/104752655-c8cbfa80-570b-11eb-933c-100baa8a4d19.png)

**Comment by UnHumbleBen** [comment]
The good news is that this doesn't seem to happen for production.
Test production code with `webdev serve --release`.
Based on this, I am guessing that the issue has to do with how Firefox retains debugging information for the Developer Tool. When in production mode, most of the debugging variables has been optimized away. I will need to examine more to figure out exactly what's causing the memory issue, but fortunately, this bug shouldn't have issue on user end

**Comment by UnHumbleBen** [comment]
@dave-doty can you still produce this issue? I cannot reproduce on Firefox 95.0.1

**Comment by dave-doty** [comment]
Yes, I have to drag it fairly fast, but I can still reproduce it. I think it only happens when you drag over objects like strands:

![image](https://user-images.githubusercontent.com/19274365/149186793-ca5c301a-78e4-4008-ac98-cff560272041.png)
